plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2844,7 +2844,8 @@ void producer_plugin_impl::schedule_production_loop() {
       // we failed to start a block, so try again later?
       _timer.async_wait([this, cid = ++_timer_corelation_id](const boost::system::error_code& ec) {
          if (ec != boost::asio::error::operation_aborted && cid == _timer_corelation_id) {
-            chain_plug->chain().interrupt_apply_block_transaction();
+            if (_is_savanna_active) // interrupt during transition causes issues, so only allow after transition
```

Comment:
Can you move this check `if (_is_savanna_active)` inside `interrupt_transaction()` so you don't do it everywhere?

Reply:
https://github.com/AntelopeIO/spring/pull/1330/commits/b5949aae9e5c97947e0d9cdb5b988a9af555862e